src/cli.rs

```diff
@@ -121,6 +126,19 @@ pub struct StreamIdGenerateArgs {
     pub r#type: StreamType,
 }
 
+#[derive(Args, Debug, Clone)]
+pub struct StreamCreateArgs {
+    /// Stream type.
+    #[arg(long, value_enum)]
+    pub r#type: StreamType,
+    /// Controller, if not set generates random value.
+    #[arg(long)]
+    pub controller: Option<String>,
+    /// Deterministic stream creation. If disabled, a "unique" field will be added to the genesis commit header.
+    #[arg(short, long, default_value_t = false)]
+    pub deterministic: bool,
```

Comment:
Could we maybe calls this `unique`? And invert its logic? So it reads When true a unique field is added to the header?